app/views/users/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<% provide :title, t(:sign_up) %>
+<h1><%= t(:sign_up) %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user, url: signup_path do |f| %>
+      <% if @user.errors.any? %>
+        <div id="error_explanation">
+          <div class="alert alert-danger">
+            <%= t (:the_form_contains)%>
+            <%= pluralize @user.errors.count, t(:error) %>.
+          </div>
+          <ul>
+          <% @user.errors.full_messages.each do |msg| %>
```

Comment:
wrong indent

Reply:
Hi Miss. Actually I cannot find which one is the wrong indent. I've tried refer my sample app but it looks similar.

Reply:
```suggestion
            <% @user.errors.full_messages.each do |msg| %>
              <li><%= msg %></li>
            <% end %>
```